.github/workflows/qa.yml

```
@@ -0,0 +1,63 @@
+name: Quality Assurance
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - master
```

```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Maybe this is the reason why the builds are nog being executed?